docs/apm/correlations.asciidoc

```diff
@@ -0,0 +1,71 @@
+Surfaced attributes are user-defined, meaning that they are completely customizable to your APM data.
+Find something interesting? A quick click of a button will auto-query your data as you work to resolve the underlying issue.
+
+For example, a site reliability engineer, who is responsible for keeping production systems up and running,
+notices an increase in latency in certain transactions.
+Analyzing metadata or tags that exist in high-latency transactions but not in lower-latency transactions
+can potentially point towards the root cause.
+They may find that a particular piece of hardware, like a host or pod, has failed, increasing latency.
+Or, perhaps a set of users, based on IP address or region, is physically too far away from the nearest
+data center, increasing latency.
+
+[discrete]
+[[view-correlations]]
+===  View correlations
+
+With a service selected, click **View correlations**:
+
+[role="screenshot"]
+image::apm/images/correlations.png[Correlations]
+
+Queries within the APM app apply to the correlations shown in the correlations fly-out.
+
+[discrete]
+[[correlations-latency]]
+====  Latency
+
+Correlations help you discover which fields are contributing to increased service latency.
+
+A latency distribution chart visualizes the overall latency of the selected service's transactions.
+Correlated attributes are sorted by _Impact_–a visual representation of the score for the underlying
+aggregation that powers correlations.
+Attributes with a high impact, or attributes present in a large percentage of slow transactions,
+may be contributing factors to increased latency.
+
+If a certain field or value seems noteworthy, use the **Filter** quick links:
+
+* `+` creates a new query in the APM app for transactions containing the selected value.
+* `-` creates a new query in the APM app for transactions without the selected value.
+
+[discrete]
+[[correlations-error-rate]]
+====  Error rate
+
+Correlations help you discover which fields are contributing to failed transactions.
+
+The Error rate over time chart visualizes the change in error rate over the selected time frame.
+Correlated attributes are sorted by _Impact_–a visual representation of the score for the underlying
+aggregation that powers correlations.
+Attributes with a high impact, or attributes present in a large percentage of failed transactions,
+may be contributing factors to increased error rates.
+
+If a certain field or value seems noteworthy, use the **Filter** quick links:
+
+* `+` creates a new query in the APM app for transactions containing the selected value.
+* `-` creates a new query in the APM app for transactions without the selected value.
+
+
+[discrete]
+[[correlations-custom-fields]]
+====  Customize fields
+
+Correlations are only as good as the data they're searching for.
+By default, a handful of potentially useful fields are selected, like `lables`, `service.version`, and `host.ip`.
```

Comment:
Lets mention all the fields we use for both frontend and backend agents in the default attributes @ogupte